### ISSUE TITLE
The POM for com.brannonsmith:MacResourceForkManager:jar:1.0.0 is missing

### ISSUE THREAD

**Issue by dmm92** [ErikaRedmark/monkey-shines-java-port#95]
Can't build with Maven from master source right now (OSX)

`$ mvn install
[INFO] Scanning for projects...
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building Monkey Shines 1.0.0
[INFO] ------------------------------------------------------------------------
Downloading: file:///Users/dmm/monkey-shines-java-port/Monkey Shines/maven_repo/com/brannonsmith/MacResourceForkManager/1.0.0/MacResourceForkManager-1.0.0.pom
[WARNING] The POM for com.brannonsmith:MacResourceForkManager:jar:1.0.0 is missing, no dependency information available
Downloading: file:///Users/dmm/monkey-shines-java-port/Monkey Shines/maven_repo/com/brannonsmith/MacResourceForkManager/1.0.0/MacResourceForkManager-1.0.0.jar
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 0.497 s
[INFO] Finished at: 2016-04-28T13:15:14-04:00
[INFO] Final Memory: 9M/309M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project monkey_shines: Could not resolve dependencies for project org.erikaredmark:monkey_shines:jar:1.0.0: Could not find artifact com.brannonsmith:MacResourceForkManager:jar:1.0.0 in monkeyshines_local_repo (file:///Users/dmm/monkey-shines-java-port/Monkey Shines/maven_repo) -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException`

**Comment by ErikaRedmark** [comment]
I believe some of those libraries I installed manually on my local machine for Maven to use. I would assume only the ones that weren't on Maven central would require installation (so Guava shouldn't cause an issue).

http://gbsmith.freeshell.org/ResCafe/#Download

This is where I got it from. It's needed for the level editor translator for old worlds.

https://maven.apache.org/guides/mini/guide-3rd-party-jars-local.html

This is how I installed the library onto my local instance so I could use Maven to compile.
